cfme/common/vm.py

```diff
@@ -792,7 +793,16 @@ def create_on_provider(self, timeout=900, find_in_cfme=False, delete_on_failure=
             find_in_cfme: Verifies that VM exists in CFME UI
             delete_on_failure: Attempts to remove VM on UI navigation failure
         """
-        vm = deploy_template(self.provider.key, self.name, self.template_name, **kwargs)
+        try:
+            import re
+            assignee = re.findall(r'assignee: *\w+',
```

Comment:
Would it be possible to use the `polarion-docstrings` library to get this data? We could potentially use `polarion_data = parser(request.function)`, and then just have `assignee = polarion_data.assignee`. https://gitlab.com/mkourim/polarion_docstrings/blob/master/polarion_docstrings/parser.py

Reply:
Here's a small example, after importing `DocstringParser` from `polarion_docstring.parser` 
```python,
In[17]: DocstringParser(test_alert_hardware_reconfigured.__doc__).parse()["assignee"].value
Out[17]: 'jdupuy'
```

Reply:
@john-dupuy I updated this.